### PR TITLE
[10.0][FIX] product_default_image: Improve performance

### PR DESCRIPTION
**Pull request by woodbrettm**
### Changes

* Stop writes to product images being called if res.company product img or target unchanged.

### CODE REVIEW

**Comment by woodbrettm** [comment]
@lasley

**Comment by lasley** [comment]
@bmw95 Add the manual SQL query to roadmap and create a ticket in +2 sprints for me to do.

**Comment by woodbrettm** [comment]
Updated

File: product_default_image/models/product_template.py [hunk 1]
```diff
@@ -256,12 +256,16 @@ def _change_template_image(self, to_type, to_img_bg=None, in_cache=False):
 
         write_map = defaultdict(lambda: self.env['product.template'].browse())
 
-        if to_img_bg:
+        if to_img_bg is not None:
+
+            if to_img_bg is False and to_type != NONE:
+                to_type = NONE
```
Comment (lasley) [inline product_default_image/models/product_template.py:262]: Please add a test for this

**Review by lasley — APPROVED** [review]
Minor comment, thanks @bmw95

**Comment by woodbrettm** [comment]
Updated. Runbot is showing a bad gateway however.

**Comment by woodbrettm** [comment]
@lasley all passing!

**Review by lasley — APPROVED** [review]

**Pull request closed without merging**